tools/common/Assembly.cs

```diff
@@ -335,6 +335,12 @@ public void ComputeLinkerFlags ()
 				foreach (var mr in m.ModuleReferences) {
 					string name = mr.Name;
 					string file = Path.GetFileNameWithoutExtension (name);
+#if MONOMAC
+					string path = Path.GetDirectoryName (name);
+					if (!path.StartsWith ("/System/Library/Frameworks"))
+						continue;
```

Comment:
This might break, in the future, if some case (e.g. `__Internal`) adds some logic - and might get unnoticed by XM/mmp.

Maybe that would be better in the `default` ?